app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt

```diff
@@ -1911,13 +2013,12 @@ class StateFragmentTest {
       rotateToLandscape()
 
       it.onActivity {
-        val mathExpressionInteractionView =
+        val mathExpressionInteractionsView =
           it.findViewById<MathExpressionInteractionsView>(
             R.id.math_expression_input_interaction_view
           )
-        assertThat(mathExpressionInteractionView.text.toString()).isEqualTo("a")
+        assertThat(mathExpressionInteractionsView.text.toString()).isEmpty()
       }
-      onView(withId(R.id.math_expression_input_error)).check(matches(isDisplayed()))
```

Comment:
Why is this being removed? Don't we need to verify that the error is retained across config changes?

Reply:
I am not able to see this conversation in PR, can you share the the test name @BenHenning

Reply:
This is ``testStateFragment_onSubmitTimeError_retainStateOnConfigChange`` @vrajdesai78.

Reply:
Updated test, Thanks @BenHenning